script.js

```diff
@@ -1,12 +1,218 @@
-//You can edit ALL of the code here
+const mainElem = document.getElementById("main-container");
+const showDropDown = document.getElementById("show-dropdown");
+const searchWrapper = document.getElementById("search-wrapper");
+const searchBar = document.getElementById("searchBar");
+const showSearchBar = document.getElementById("searchShow");
+const select = document.createElement("select");
+const seeAllElements = document.createElement("option");
+const paragraph = document.getElementById("display-text");
+const homeButton = document.getElementById("button");
+let allShows = getAllShows();
+
+// function to display all shows on when page loads
+function displayShows(allShows) {
+    showsInformation(allShows);
+    document.querySelectorAll(".show-container").forEach((show) => {
+        let showId = show.getAttribute("data-id");
```

Comment:
I'm new to this getAttribute ("data-id"). I assume this is referencing the id property of each show. nice!